python/lsst/ap/pipe/metrics.py

```diff
@@ -26,10 +26,13 @@
 __all__ = [
     "ApFakesCompletenessMetricTask", "ApFakesCompletenessMetricConfig",
     "ApFakesCountMetricTask", "ApFakesCountMetricConfig",
+    "ApFakesCompletenessSNRMetricTask", "ApFakesCompletenessSNRMetricConfig",
+    "ApFakesCountSNRMetricTask", "ApFakesCountSNRMetricConfig",
```

Comment:
If these are implemented as analysis_tools, we probably don't also want them here.